docs/GenericsManifesto.md

```diff
@@ -87,6 +87,21 @@ typealias StringDictionary<Value> = Dictionary<String, Value>
 var d1 = StringDictionary<Int>()
 var d2: Dictionary<String, Int> = d1 // okay: d1 and d2 have the same type, Dictionary<String, Int>
 ```
+### Generic associatedtypes
```

Comment:
"associated types" here and below

Reply:
That’s how I would usually write it but I followed the precedent of “typealiases” that’s already in this document.

Reply:
I think that should be fixed too. In diagnostics we tend to say "type alias" not "typealias"

Reply:
Do you suggest I include fixes for all of them in this PR?

Reply:
@DougGregor any thoughts on this?  Should I make the change from "associatedtypes"  to "associated types" and "typealiases" to "type aliases" in this PR?

Reply:
It's probably worth a small note about how this might look from the perspective of somebody trying to conform to a protocol with generic associated types. The way I guess it would work is that a generic associated-type would require a generic type alias in the conforming type. So the generic type which fulfils that requirement may have more generic parameters, as long as they are all bound to parameters in the parent type or to concrete types.

So in your example, `Wrapped<T>` could be a tuple of `(Int, T)`, or an `Array<T>`, or a `Dictionary<SomeNestedGenericParameter, T>`. Is that correct?

Reply:
@karwa This is a good idea.  I'll be updating the PR with a note about this shortly.  Your understanding is correct.